src/frontend/components/feedbackBoardMetadataFormPermissions.tsx

```diff
@@ -73,9 +83,9 @@ function FeedbackBoardMetadataFormPermissions(props: Readonly<IFeedbackBoardMeta
 
       return o.name.toLowerCase().includes(newSearchTerm.toLowerCase());
     });
-
+    const deduplicatedOptions = deduplicatePermissionOptions(filteredOptions);
```

Comment:
Can we simplify the code and reduce the state usage?

_I didn't test the suggestion below, but what I mean is something like this (from top of my mind)_

```suggestion
    const deduplicatedOptions = Array.from(new Map(filteredOptions.map(option => [option.id, option])).values())
```

Reply:
I removed the function deduplicatePermissionOptions, and changed logic to use what you suggested above. Also only use it in one place now instead of three.